demos/vc_issuer/README.md

```diff
@@ -0,0 +1,15 @@
+# An example issuer of verifiable credentials
+
+This is an example implementation of an issuer of verifiable credentials  for 
+the [attribute sharing flow on the IC](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/attribute-sharing.md).
+
+## Supported Behavior
+
+The app implements the interface of the issuer from [the spec](../../docs/vc-spec.md), plus some additional
+APIs for configuring and testing, cf. [vc_issuer.did](./vc_issuer.did).
+Please note that this issuer is for demonstrating the use of issuer APIS, and is **not meant 
+for real-world deployment**, as it does not have proper management of user data.
+
+## Development
+
+Run `./build.sh` script to build the issuer canister.
```

Comment:
worth adding notes about how to initialize it? I have this in my notes, some of it courtesy of @frederikrothenberger :


```
$ ii_canister_id=$(cat .dfx/local/canister_ids.json | jq -cMr '.internet_identity.local')
$ rootkey=$(curl -s http://localhost/:$(dfx info replica-port)/api/v2/status | npx --package=cbor-cli -- cbor2json | jq -cM '.value.root_key.data' | sed 's/\[/{/g;s/,/;/g;s/\]/}/g')
~/internet-identity$ dfx canister call issuer configure '(record { idp_canister_ids = vec{ principal "'$ii_canister_id'" }; ic_root_key_der = vec '"$rootkey"' })'

$ dfx canister call issuer add_employee '(principal "3d7z4-fytna-ykqnl-dzl5q-dgv5v-rxuhm-j4cf7-2mmbi-6wewg-k7syq-nae")'
```

Reply:
We'll add initialisation instructions in the coming days, once we get something nice-looking -- thanks for sharing the notes: I believe we can iterate on them to get something to commit.